scss/_functions.scss

```diff
@@ -271,3 +271,9 @@ $_luminance-list: .0008 .001 .0011 .0013 .0015 .0017 .002 .0022 .0025 .0027 .003
   }
   @return $result;
 }
+
+@mixin validate-utilities-field($utility, $utility-name, $field) {
+  @if (not map-has-key($utility, $field)) {
+    @warn "Required field «#{$field}» is empty. Check the field name for the «#{$utility-name}» utility";
```

Comment:
Use single quote to wrap variable name. You can see other `@warn` on top of this file. :)

I'd simplify the sentence to make it more obvious, maybe something like: `Missing '#{$field}' for '#{$utility-name}' utility.`. Any thought?

Reply:
Yes good idea